kubernetes/annotations_and_labels.md

```diff
@@ -51,16 +47,38 @@ This page defines common annotations and labels we set in Kubernetes objects.
   value may be equal to that of `app.kubernetes.io/version` but it has a
   different purpose and since there could be multiple operators reconciling one
   object there could be multiple per-operator labels on one object.
-- `app.giantswarm.io/branch` - (informational) branch from which this
-  instance of the app/operator was built from.
+- `helm.sh/chart` - value should contain a chart name and version, see
+  [Helm chart labels best practices][helm-labels]
+- `app.kubernetes.io/name` - the name of the application; should be applied to
+  every Kubernetes resource associated with that application, i.e. all objects
+  in a chart that installs it.
```

Comment:
It would be cool to add some example here.

Reply:
Done